Editor updates (#6001)
mintlify/docs: editor/collaborate.mdx

- Undo only affects your own changes—undoing won't revert another person's edits.
- If you lose your network connection, your edits save locally and sync when you reconnect.

## Chat with the agent in the editor

The agent panel lets you chat with the [Mintlify agent](/agent) directly inside the editor. Open it by clicking the agent icon in the left sidebar. The panel sits to the left of the file explorer so you can chat and navigate files at the same time.

### Start a session

Type a prompt in the chat input and press Enter. The agent reads your docs and makes edits on the current branch.

### Attach context

Click the paperclip icon in the chat input to attach context to your message. You can attach:

- **Files** — any file from your repository
- **Pages** — specific documentation pages the agent should read or edit
- **Divisions** — a tab, anchor, or other navigation section
- **Inline text** — paste text directly into the attachment field

Attaching context focuses the agent on the right part of your docs without requiring you to describe it in your prompt.

### Review per-session diffs

> After the agent makes changes, a diff summary appears in the chat panel showing exactly what was added, removed, or rewritten. Click any file in the diff summary to jump directly to that page in the editor.

Author: ```suggestion
After the agent makes changes, a diff summary appears in the chat panel showing the changes. Click any file in the diff summary to navigate to that page in the editor.
```

### Leave feedback

> Use the thumbs up / thumbs down controls in the chat panel to rate the agent's response. This helps improve future results.

Author: ```suggestion
Use the thumbs up and thumbs down buttons in the chat panel to rate the agent's response. This helps improve future results.
```

The agent's cursor clears automatically a short time after the agent finishes editing.

## Send content to the assistant

> To send highlighted content to the [assistant](/assistant), select text in the editor — a floating toolbar appears above the selection — then click **Add to Assistant**. The selected content is attached to a new Assistant conversation so you can ask questions or request changes without switching context.

Author: ```suggestion
To send highlighted content to the [assistant](/assistant), select text in the editor. A floating toolbar appears above the selection. Click **Add to Assistant**. The selected content attaches to a new assistant conversation so you can ask questions or request changes without switching context.
```
